ONSdigital/census-rm-ddl
# Added backout scripts to be used for CE sample load

# Motivation and Context
We are doing our first sample load into production (CEs & SPGs).

# What has changed
Added backout scripts

# How to test?
1. Load an unaddressed batch
2. Load a sample
3. Run the backout scripts
4. Check that the unaddressed uac-qids (and associated events) are untouched
5. Check that the cases and associated events and uac-qids, have been deleted
6. Do a little victory dance

# Links
Trello: https://trello.com/c/0swcWcaa
